liquibase-couchbase/src/test/java/integration/statement/DropPrimaryIndexStatementIT.java

```diff
@@ -50,9 +45,9 @@ void Should_drop_Primary_index_by_name() {
     void Should_drop_primary_index_for_specific_keyspace() {
         cluster.waitUntilReady(CLUSTER_READY_TIMEOUT);
         keyspace = keyspace(bucketName, scopeName, collectionName);
-        Collection collection = clusterOperator.getBucketOperator(bucketName)
-                .getCollection(collectionName, scopeName);
-        clusterOperator.getCollectionOperator(collection).createCollectionPrimaryIndex(null);
+        clusterOperator.getBucketOperator(bucketName)
+                .getCollectionOperator(collectionName, scopeName)
+                .createCollectionPrimaryIndex(null);
```

Comment:
I think it should be `createPrimaryIndex()` method call as it plays the same role as `.createCollectionPrimaryIndex(null)`